2023-05-15 11:35:58.097 WARN [Thread-5][AbstractBootstrap.java:460] - Unknown channel option 'TCP_NODELAY' for channel '[id: 0x9edc70f3]'
2023-05-15 11:35:58.097 WARN [Thread-5][AbstractBootstrap.java:460] - Unknown channel option 'SO_KEEPALIVE' for channel '[id: 0x9edc70f3]'
2023-05-15 11:35:58.381 WARN [nioEventLoopGroup-2-1][ChannelInitializer.java:97] - Failed to initialize a channel. Closing: [id: 0x7b77b048]
java.lang.IllegalStateException: SSLContextImpl is not initialized
	at sun.security.ssl.SSLContextImpl.engineCreateSSLEngine(SSLContextImpl.java:196) ~[?:1.8.0_221]
	at javax.net.ssl.SSLContext.createSSLEngine(SSLContext.java:329) ~[?:1.8.0_221]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:72) ~[classes/:?]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:1) ~[classes/:?]
	at io.netty.channel.ChannelInitializer.initChannel(ChannelInitializer.java:129) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.ChannelInitializer.handlerAdded(ChannelInitializer.java:112) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannelHandlerContext.callHandlerAdded(AbstractChannelHandlerContext.java:964) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAdded0(DefaultChannelPipeline.java:610) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.access$100(DefaultChannelPipeline.java:46) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline$PendingHandlerAddedTask.execute(DefaultChannelPipeline.java:1461) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAddedForAllHandlers(DefaultChannelPipeline.java:1126) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.invokeHandlerAddedIfNeeded(DefaultChannelPipeline.java:651) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:515) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:428) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:487) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:405) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:906) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_221]
2023-05-15 11:35:58.384 ERROR [nioEventLoopGroup-2-1][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:36:01.097 WARN [nioEventLoopGroup-2-2][ChannelInitializer.java:97] - Failed to initialize a channel. Closing: [id: 0xf1dd61c6]
java.lang.IllegalStateException: SSLContextImpl is not initialized
	at sun.security.ssl.SSLContextImpl.engineCreateSSLEngine(SSLContextImpl.java:196) ~[?:1.8.0_221]
	at javax.net.ssl.SSLContext.createSSLEngine(SSLContext.java:329) ~[?:1.8.0_221]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:72) ~[classes/:?]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:1) ~[classes/:?]
	at io.netty.channel.ChannelInitializer.initChannel(ChannelInitializer.java:129) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.ChannelInitializer.handlerAdded(ChannelInitializer.java:112) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannelHandlerContext.callHandlerAdded(AbstractChannelHandlerContext.java:964) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAdded0(DefaultChannelPipeline.java:610) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.access$100(DefaultChannelPipeline.java:46) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline$PendingHandlerAddedTask.execute(DefaultChannelPipeline.java:1461) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAddedForAllHandlers(DefaultChannelPipeline.java:1126) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.invokeHandlerAddedIfNeeded(DefaultChannelPipeline.java:651) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:515) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:428) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:487) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:405) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:906) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_221]
2023-05-15 11:36:01.097 ERROR [nioEventLoopGroup-2-2][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:36:04.112 WARN [nioEventLoopGroup-2-3][ChannelInitializer.java:97] - Failed to initialize a channel. Closing: [id: 0x116eaed0]
java.lang.IllegalStateException: SSLContextImpl is not initialized
	at sun.security.ssl.SSLContextImpl.engineCreateSSLEngine(SSLContextImpl.java:196) ~[?:1.8.0_221]
	at javax.net.ssl.SSLContext.createSSLEngine(SSLContext.java:329) ~[?:1.8.0_221]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:72) ~[classes/:?]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:1) ~[classes/:?]
	at io.netty.channel.ChannelInitializer.initChannel(ChannelInitializer.java:129) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.ChannelInitializer.handlerAdded(ChannelInitializer.java:112) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannelHandlerContext.callHandlerAdded(AbstractChannelHandlerContext.java:964) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAdded0(DefaultChannelPipeline.java:610) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.access$100(DefaultChannelPipeline.java:46) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline$PendingHandlerAddedTask.execute(DefaultChannelPipeline.java:1461) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAddedForAllHandlers(DefaultChannelPipeline.java:1126) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.invokeHandlerAddedIfNeeded(DefaultChannelPipeline.java:651) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:515) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:428) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:487) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:405) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:906) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_221]
2023-05-15 11:36:04.112 ERROR [nioEventLoopGroup-2-3][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:36:07.125 WARN [nioEventLoopGroup-2-4][ChannelInitializer.java:97] - Failed to initialize a channel. Closing: [id: 0x6a820d79]
java.lang.IllegalStateException: SSLContextImpl is not initialized
	at sun.security.ssl.SSLContextImpl.engineCreateSSLEngine(SSLContextImpl.java:196) ~[?:1.8.0_221]
	at javax.net.ssl.SSLContext.createSSLEngine(SSLContext.java:329) ~[?:1.8.0_221]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:72) ~[classes/:?]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:1) ~[classes/:?]
	at io.netty.channel.ChannelInitializer.initChannel(ChannelInitializer.java:129) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.ChannelInitializer.handlerAdded(ChannelInitializer.java:112) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannelHandlerContext.callHandlerAdded(AbstractChannelHandlerContext.java:964) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAdded0(DefaultChannelPipeline.java:610) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.access$100(DefaultChannelPipeline.java:46) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline$PendingHandlerAddedTask.execute(DefaultChannelPipeline.java:1461) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAddedForAllHandlers(DefaultChannelPipeline.java:1126) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.invokeHandlerAddedIfNeeded(DefaultChannelPipeline.java:651) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:515) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:428) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:487) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:405) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:906) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_221]
2023-05-15 11:36:07.127 ERROR [nioEventLoopGroup-2-4][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:36:10.152 WARN [nioEventLoopGroup-2-5][ChannelInitializer.java:97] - Failed to initialize a channel. Closing: [id: 0x40efcd4d]
java.lang.IllegalStateException: SSLContextImpl is not initialized
	at sun.security.ssl.SSLContextImpl.engineCreateSSLEngine(SSLContextImpl.java:196) ~[?:1.8.0_221]
	at javax.net.ssl.SSLContext.createSSLEngine(SSLContext.java:329) ~[?:1.8.0_221]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:72) ~[classes/:?]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:1) ~[classes/:?]
	at io.netty.channel.ChannelInitializer.initChannel(ChannelInitializer.java:129) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.ChannelInitializer.handlerAdded(ChannelInitializer.java:112) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannelHandlerContext.callHandlerAdded(AbstractChannelHandlerContext.java:964) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAdded0(DefaultChannelPipeline.java:610) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.access$100(DefaultChannelPipeline.java:46) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline$PendingHandlerAddedTask.execute(DefaultChannelPipeline.java:1461) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAddedForAllHandlers(DefaultChannelPipeline.java:1126) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.invokeHandlerAddedIfNeeded(DefaultChannelPipeline.java:651) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:515) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:428) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:487) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:405) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:906) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_221]
2023-05-15 11:36:10.152 ERROR [nioEventLoopGroup-2-5][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:36:13.141 WARN [nioEventLoopGroup-2-6][ChannelInitializer.java:97] - Failed to initialize a channel. Closing: [id: 0xf7eaf3f7]
java.lang.IllegalStateException: SSLContextImpl is not initialized
	at sun.security.ssl.SSLContextImpl.engineCreateSSLEngine(SSLContextImpl.java:196) ~[?:1.8.0_221]
	at javax.net.ssl.SSLContext.createSSLEngine(SSLContext.java:329) ~[?:1.8.0_221]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:72) ~[classes/:?]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:1) ~[classes/:?]
	at io.netty.channel.ChannelInitializer.initChannel(ChannelInitializer.java:129) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.ChannelInitializer.handlerAdded(ChannelInitializer.java:112) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannelHandlerContext.callHandlerAdded(AbstractChannelHandlerContext.java:964) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAdded0(DefaultChannelPipeline.java:610) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.access$100(DefaultChannelPipeline.java:46) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline$PendingHandlerAddedTask.execute(DefaultChannelPipeline.java:1461) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAddedForAllHandlers(DefaultChannelPipeline.java:1126) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.invokeHandlerAddedIfNeeded(DefaultChannelPipeline.java:651) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:515) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:428) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:487) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:405) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:906) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_221]
2023-05-15 11:36:13.141 ERROR [nioEventLoopGroup-2-6][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:36:16.154 WARN [nioEventLoopGroup-2-7][ChannelInitializer.java:97] - Failed to initialize a channel. Closing: [id: 0x38c97851]
java.lang.IllegalStateException: SSLContextImpl is not initialized
	at sun.security.ssl.SSLContextImpl.engineCreateSSLEngine(SSLContextImpl.java:196) ~[?:1.8.0_221]
	at javax.net.ssl.SSLContext.createSSLEngine(SSLContext.java:329) ~[?:1.8.0_221]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:72) ~[classes/:?]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:1) ~[classes/:?]
	at io.netty.channel.ChannelInitializer.initChannel(ChannelInitializer.java:129) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.ChannelInitializer.handlerAdded(ChannelInitializer.java:112) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannelHandlerContext.callHandlerAdded(AbstractChannelHandlerContext.java:964) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAdded0(DefaultChannelPipeline.java:610) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.access$100(DefaultChannelPipeline.java:46) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline$PendingHandlerAddedTask.execute(DefaultChannelPipeline.java:1461) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAddedForAllHandlers(DefaultChannelPipeline.java:1126) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.invokeHandlerAddedIfNeeded(DefaultChannelPipeline.java:651) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:515) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:428) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:487) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:405) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:906) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_221]
2023-05-15 11:36:16.154 ERROR [nioEventLoopGroup-2-7][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:36:19.156 WARN [nioEventLoopGroup-2-8][ChannelInitializer.java:97] - Failed to initialize a channel. Closing: [id: 0x36af5f9b]
java.lang.IllegalStateException: SSLContextImpl is not initialized
	at sun.security.ssl.SSLContextImpl.engineCreateSSLEngine(SSLContextImpl.java:196) ~[?:1.8.0_221]
	at javax.net.ssl.SSLContext.createSSLEngine(SSLContext.java:329) ~[?:1.8.0_221]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:72) ~[classes/:?]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:1) ~[classes/:?]
	at io.netty.channel.ChannelInitializer.initChannel(ChannelInitializer.java:129) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.ChannelInitializer.handlerAdded(ChannelInitializer.java:112) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannelHandlerContext.callHandlerAdded(AbstractChannelHandlerContext.java:964) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAdded0(DefaultChannelPipeline.java:610) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.access$100(DefaultChannelPipeline.java:46) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline$PendingHandlerAddedTask.execute(DefaultChannelPipeline.java:1461) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAddedForAllHandlers(DefaultChannelPipeline.java:1126) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.invokeHandlerAddedIfNeeded(DefaultChannelPipeline.java:651) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:515) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:428) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:487) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:405) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:906) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_221]
2023-05-15 11:36:19.156 ERROR [nioEventLoopGroup-2-8][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:36:22.168 WARN [nioEventLoopGroup-2-9][ChannelInitializer.java:97] - Failed to initialize a channel. Closing: [id: 0xab4ef812]
java.lang.IllegalStateException: SSLContextImpl is not initialized
	at sun.security.ssl.SSLContextImpl.engineCreateSSLEngine(SSLContextImpl.java:196) ~[?:1.8.0_221]
	at javax.net.ssl.SSLContext.createSSLEngine(SSLContext.java:329) ~[?:1.8.0_221]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:72) ~[classes/:?]
	at com.bugbycode.forward.client.StartupRunnable$1.initChannel(StartupRunnable.java:1) ~[classes/:?]
	at io.netty.channel.ChannelInitializer.initChannel(ChannelInitializer.java:129) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.ChannelInitializer.handlerAdded(ChannelInitializer.java:112) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannelHandlerContext.callHandlerAdded(AbstractChannelHandlerContext.java:964) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAdded0(DefaultChannelPipeline.java:610) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.access$100(DefaultChannelPipeline.java:46) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline$PendingHandlerAddedTask.execute(DefaultChannelPipeline.java:1461) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAddedForAllHandlers(DefaultChannelPipeline.java:1126) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.DefaultChannelPipeline.invokeHandlerAddedIfNeeded(DefaultChannelPipeline.java:651) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:515) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:428) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:487) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:405) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:906) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-all-4.1.36.Final.jar:4.1.36.Final]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_221]
2023-05-15 11:36:22.168 ERROR [nioEventLoopGroup-2-9][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:36:43.783 WARN [Thread-5][AbstractBootstrap.java:460] - Unknown channel option 'TCP_NODELAY' for channel '[id: 0x8dc13958]'
2023-05-15 11:36:43.783 WARN [Thread-5][AbstractBootstrap.java:460] - Unknown channel option 'SO_KEEPALIVE' for channel '[id: 0x8dc13958]'
2023-05-15 11:37:07.378 ERROR [nioEventLoopGroup-2-1][ClientHandler.java:105] - 远程主机强迫关闭了一个现有的连接。
2023-05-15 11:37:09.086 ERROR [nioEventLoopGroup-2-3][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:37:38.736 WARN [Thread-5][AbstractBootstrap.java:460] - Unknown channel option 'TCP_NODELAY' for channel '[id: 0x0c4f974b]'
2023-05-15 11:37:38.737 WARN [Thread-5][AbstractBootstrap.java:460] - Unknown channel option 'SO_KEEPALIVE' for channel '[id: 0x0c4f974b]'
2023-05-15 11:41:13.218 ERROR [nioEventLoopGroup-2-13][ClientHandler.java:105] - 远程主机强迫关闭了一个现有的连接。
2023-05-15 11:41:16.405 ERROR [nioEventLoopGroup-2-14][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:41:19.386 ERROR [nioEventLoopGroup-2-15][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:41:22.372 ERROR [nioEventLoopGroup-2-16][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:41:25.437 ERROR [nioEventLoopGroup-2-1][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:41:28.414 ERROR [nioEventLoopGroup-2-2][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:41:31.384 ERROR [nioEventLoopGroup-2-3][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:41:34.471 ERROR [nioEventLoopGroup-2-4][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:41:37.419 ERROR [nioEventLoopGroup-2-5][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:41:40.442 ERROR [nioEventLoopGroup-2-6][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:41:43.475 ERROR [nioEventLoopGroup-2-7][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:41:46.488 ERROR [nioEventLoopGroup-2-8][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:41:49.489 ERROR [nioEventLoopGroup-2-9][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:41:52.424 ERROR [nioEventLoopGroup-2-10][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:41:55.455 ERROR [nioEventLoopGroup-2-11][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:41:58.474 ERROR [nioEventLoopGroup-2-12][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:42:01.506 ERROR [nioEventLoopGroup-2-13][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:42:04.476 ERROR [nioEventLoopGroup-2-14][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:42:07.507 ERROR [nioEventLoopGroup-2-15][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:42:10.554 ERROR [nioEventLoopGroup-2-16][StartupRunnable.java:95] - Connection to localhost:36500 failed...
2023-05-15 11:42:33.815 ERROR [nioEventLoopGroup-4-9][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:42:33.986 ERROR [nioEventLoopGroup-4-12][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:42:33.990 ERROR [nioEventLoopGroup-4-13][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:43:49.682 ERROR [nioEventLoopGroup-4-14][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:43:49.683 ERROR [nioEventLoopGroup-4-14][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:43:49.965 ERROR [nioEventLoopGroup-4-1][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:26.398 ERROR [nioEventLoopGroup-4-2][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:26.399 ERROR [nioEventLoopGroup-4-2][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:26.403 ERROR [nioEventLoopGroup-4-3][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:26.408 ERROR [nioEventLoopGroup-4-4][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:26.418 ERROR [nioEventLoopGroup-4-5][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:26.473 ERROR [nioEventLoopGroup-4-6][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:26.893 ERROR [nioEventLoopGroup-4-7][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:27.100 ERROR [nioEventLoopGroup-4-8][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:27.229 ERROR [nioEventLoopGroup-4-9][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:27.337 ERROR [nioEventLoopGroup-4-10][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:32.188 ERROR [nioEventLoopGroup-4-13][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:32.188 ERROR [nioEventLoopGroup-4-13][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:32.205 ERROR [nioEventLoopGroup-4-14][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:48.466 ERROR [nioEventLoopGroup-4-16][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:48.467 ERROR [nioEventLoopGroup-4-16][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:49.117 ERROR [nioEventLoopGroup-4-1][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:49.117 ERROR [nioEventLoopGroup-4-1][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:49.333 ERROR [nioEventLoopGroup-4-2][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:49.334 ERROR [nioEventLoopGroup-4-2][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:49.337 ERROR [nioEventLoopGroup-4-3][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:50.011 ERROR [nioEventLoopGroup-4-4][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:50.068 ERROR [nioEventLoopGroup-4-5][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:50.725 ERROR [nioEventLoopGroup-4-11][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:50.914 ERROR [nioEventLoopGroup-4-6][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:51.336 ERROR [nioEventLoopGroup-4-12][AgentHandler.java:298] - Connection to content-autofill.googleapis.com:443 failed.
2023-05-15 11:44:53.370 ERROR [nioEventLoopGroup-4-7][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:53.371 ERROR [nioEventLoopGroup-4-7][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:53.597 ERROR [nioEventLoopGroup-4-8][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:58.301 ERROR [nioEventLoopGroup-4-9][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:58.302 ERROR [nioEventLoopGroup-4-9][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:44:58.306 ERROR [nioEventLoopGroup-4-10][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:03.424 ERROR [nioEventLoopGroup-4-12][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:03.424 ERROR [nioEventLoopGroup-4-12][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:03.424 ERROR [nioEventLoopGroup-4-13][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:14.652 ERROR [nioEventLoopGroup-4-14][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:14.652 ERROR [nioEventLoopGroup-4-14][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:14.652 ERROR [nioEventLoopGroup-4-15][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:23.981 ERROR [nioEventLoopGroup-4-16][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:23.981 ERROR [nioEventLoopGroup-4-16][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:23.998 ERROR [nioEventLoopGroup-4-1][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:26.446 ERROR [nioEventLoopGroup-4-2][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:26.446 ERROR [nioEventLoopGroup-4-2][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:26.446 ERROR [nioEventLoopGroup-4-3][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:31.647 ERROR [nioEventLoopGroup-4-4][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:31.647 ERROR [nioEventLoopGroup-4-4][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:31.647 ERROR [nioEventLoopGroup-4-5][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:36.932 ERROR [nioEventLoopGroup-4-6][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:36.948 ERROR [nioEventLoopGroup-4-6][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:36.948 ERROR [nioEventLoopGroup-4-7][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:36.948 ERROR [nioEventLoopGroup-4-8][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:48.158 ERROR [nioEventLoopGroup-4-9][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:48.158 ERROR [nioEventLoopGroup-4-9][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:48.158 ERROR [nioEventLoopGroup-4-10][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:57.413 ERROR [nioEventLoopGroup-4-11][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:57.414 ERROR [nioEventLoopGroup-4-11][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:57.415 ERROR [nioEventLoopGroup-4-12][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:59.893 ERROR [nioEventLoopGroup-4-13][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:59.893 ERROR [nioEventLoopGroup-4-13][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:45:59.909 ERROR [nioEventLoopGroup-4-14][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:46:05.027 ERROR [nioEventLoopGroup-4-15][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:46:05.027 ERROR [nioEventLoopGroup-4-15][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:46:05.027 ERROR [nioEventLoopGroup-4-16][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:46:10.025 ERROR [nioEventLoopGroup-4-1][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:46:10.025 ERROR [nioEventLoopGroup-4-1][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:46:10.025 ERROR [nioEventLoopGroup-4-2][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:46:21.149 ERROR [nioEventLoopGroup-4-3][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:46:21.149 ERROR [nioEventLoopGroup-4-3][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:46:21.149 ERROR [nioEventLoopGroup-4-4][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:06.337 WARN [Thread-5][AbstractBootstrap.java:460] - Unknown channel option 'TCP_NODELAY' for channel '[id: 0x8a61a8bf]'
2023-05-15 11:47:06.337 WARN [Thread-5][AbstractBootstrap.java:460] - Unknown channel option 'SO_KEEPALIVE' for channel '[id: 0x8a61a8bf]'
2023-05-15 11:47:17.953 ERROR [nioEventLoopGroup-4-1][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:17.955 ERROR [nioEventLoopGroup-4-1][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:17.959 ERROR [nioEventLoopGroup-4-5][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:17.982 ERROR [nioEventLoopGroup-4-2][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:17.983 ERROR [nioEventLoopGroup-4-2][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:17.994 ERROR [nioEventLoopGroup-4-6][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:19.219 ERROR [nioEventLoopGroup-4-8][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:19.220 ERROR [nioEventLoopGroup-4-8][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:19.222 ERROR [nioEventLoopGroup-4-10][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:19.249 ERROR [nioEventLoopGroup-4-9][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:30.346 ERROR [nioEventLoopGroup-4-12][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:30.347 ERROR [nioEventLoopGroup-4-12][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:30.353 ERROR [nioEventLoopGroup-4-13][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:30.817 ERROR [nioEventLoopGroup-4-11][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:30.820 ERROR [nioEventLoopGroup-4-14][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:31.727 ERROR [nioEventLoopGroup-4-16][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:31.729 ERROR [nioEventLoopGroup-4-15][AgentHandler.java:298] - Unconnected forward server
2023-05-15 11:47:57.996 ERROR [nioEventLoopGroup-2-8][ClientHandler.java:105] - 远程主机强迫关闭了一个现有的连接。
